turbo.json

```diff
@@ -22,7 +22,7 @@
       "cache": false,
       "persistent": true
     },
-    "lint": { "dependsOn": ["compile", "^build"] },
+    "lint": { "dependsOn": ["compile"] },
```

Comment:
Now the external packages are being used as internal again, we don't need to pre-build them